cfme/tests/test_appliance.py

```diff
@@ -83,6 +83,14 @@ def test_httpd_running(ssh_client):
     assert 'is running' in stdout
 
 
+@pytest.mark.meta(blockers=[1336742])
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.5')
+def test_journald_running(ssh_client):
+    """Verifies systemd-journald service is running on the appliance"""
+    stdout = ssh_client.run_command('systemctl status systemd-journald')[1]
```

Comment:
There should be an issue raised for inconsistency and a PR fixing it, that's how it should be handled in the future, not by blocking a PR...